README.md

```diff
@@ -15,39 +12,43 @@ This extension adds rich language support for the Go language to VS Code, includ
 - Quick Info (using `gogetdoc` or `godef`+`godoc`)
 - Goto Definition (using `gogetdoc` or `godef`+`godoc`)
 - Find References (using `guru`)
+- Find implementations (using `guru`)
```

Comment:
Merge from master, there are a lot of changes there

Reply:
I'm rebase from the master, seems not good rebase to an existing PR

Reply:
Do you want me to re-create clean PR?

Reply:
Yes please, let's just do that. There is too much noise this PR.